src/main/java/com/denizenscript/denizen2core/tags/objects/QueueTag.java

```diff
@@ -135,9 +127,7 @@ public double getNumberForm() {
         // @Returns the value of the specified definition on the queue.
         // @Example "1" .definition[value] may return "3".
         // -->
-        handlers.put("definition", (dat, obj) -> {
-            return ((QueueTag) obj).internal.commandStack.peek().getDefinition(dat.getNextModifier().toString());
-        });
+        handlers.put("definition", (dat, obj) -> ((QueueTag) obj).internal.commandStack.peek().getDefinition(dat.getNextModifier().toString()));
```

Comment:
Oh wow that's old code here! Need to patch this (and any similar code) to account for risk of it returning an internal null. Should return a NullTag or an error in those cases.